controller_name: 'IdealDiffDriveMPC'
rate: 20
minimum_linear_velocity: 0.1
maximum_linear_velocity: 3.0
maximum_angular_velocity: 1.0
goal_tolerance: 0.2
gain_distance_to_goal_linear: 0.5
path_look_ahead_distance: 1.0
query_radius: 50
query_knn: 10
id_window_size: 5
horizon_length: 30
state_cost_translational: 1
state_cost_rotational: 0.0
input_cost_wheel: 0.05
angular_velocity_gain: 1.3
wheel_radius: 0.165
baseline: 0.55
